pyeudiw/tests/satosa/test_backend.py

```diff
@@ -45,6 +49,12 @@
         "logo_path": None,
         "use_zlib": True
     },
+    "sd_jwt_settings": {
+        "issuer": "http://test.com", 
+        "default_exp": 60, 
+        "specification_file": "./pyeudiw/tests/tools/specifications.yml", 
```

Comment:
please take the specification def from the backend config and not from the filesystem